pallet/staking/src/lib.rs

```diff
@@ -1329,9 +1329,10 @@ where
 				.unwrap_or_default(),
 			signature,
 		};
-		// b"sc/ktstk"
-		let sender =
-			H160([115, 99, 47, 107, 116, 115, 116, 107, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]);
+		// Treasury account.
+		let sender = H160([
+			109, 111, 100, 108, 100, 97, 47, 116, 114, 115, 114, 121, 0, 0, 0, 0, 0, 0, 0, 0,
+		]);
```

Comment:
Can you add a comment to attach the readable address? Make it easy to search the account information in the explorer.

Reply:
I made a quite big refactor PR. We can fix this there.